content/zh/docs/project-user-guide/application-workloads/routes.md

```diff
@@ -0,0 +1,178 @@
+   * **名称**：应用路由的名称，用作此应用路由的唯一标识符。
+   * **别名**：应用路由的别名。
+   * **描述信息**：应用路由的描述信息。
+
+   ![basic-info](/images/docs/zh-cn/project-user-guide/application-workloads/routes/basic-info.png)
+
+### 步骤 2：配置路由规则
+
+1. 在**路由规则**选项卡中，点击**添加路由规则**。
+
+2. 选择一种模式来配置路由规则，点击 **√**，然后点击**下一步**。
+
+   * **自动生成**：KubeSphere 自动以`<服务名称>.<项目名称>.<网关地址>.nip.io` 格式生成域名，该域名由 [nip.io](https://nip.io/) 自动解析为网关地址。该模式仅支持 HTTP。
+     
+     * **路径**：将每个服务映射到一条路径。您可以点击**添加 Path** 来添加多条路径。
+     
+     ![auto-generate](/images/docs/zh-cn/project-user-guide/application-workloads/routes/auto-generate.png)
+     
+   * **指定域名**：使用用户定义的域名。此模式同时支持 HTTP 和 HTTPS。
+     
+     * **域名**：为应用路由设置域名。
+     * **协议**：选择 `http` 或 `https`。如果选择了 `https`，则需要选择包含 `tls.crt`（TLS 证书）和 `tls.key`（TLS 私钥）的密钥用于加密。
+     * **路径**：将每个服务映射到一条路径。您可以点击**添加 Path** 来添加多条路径。
+   
+     ![specify-domain](/images/docs/zh-cn/project-user-guide/application-workloads/routes/specify-domain.png)
+
+### （可选）步骤 3：配置高级设置
+
+1. 在**高级设置**选项卡，选择**添加元数据**。
+
+   为应用路由配置注解和标签，并点击**创建**。
+
+   {{< notice note >}}
+
+   您可以使用注解来自定义应用路由的行为。有关更多信息，请参见 [Nginx Ingress controller 官方文档](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/)。
+
+   {{</ notice >}}
+
+   ![add-metadata](/images/docs/zh-cn/project-user-guide/application-workloads/routes/add-metadata.png)
+
+### 步骤 4：获取域名、服务路径和网关地址
+
+1. 在左侧导航栏中选择**应用负载**中的**应用路由**，点击右侧的应用路由名称。
+
+   ![route-list](/images/docs/zh-cn/project-user-guide/application-workloads/routes/route-list.png)
+
+2. 在**规则**区域获取域名和服务路径，在**详情**区域获取网关地址。
+
+   * 如果[应用路由访问方式](#configure-the-route-access-method)设置为 NodePort，则会使用 Kubernetes 集群节点的 IP 地址作为网关地址，NodePort 位于在域名之后。
+
+     ![obtain-address-nodeport](/images/docs/zh-cn/project-user-guide/application-workloads/routes/obtain-address-nodeport.png)
+
+   * 如果[应用路由访问方式](#configure-the-route-access-method)设置为 LoadBalancer，则网关地址由负载均衡器插件指定。
+
+     ![obtain-address-loadbalancer](/images/docs/zh-cn/project-user-guide/application-workloads/routes/obtain-address-loadbalancer.png)
+
+## 配置域名解析
+
+若在[配置路由规则](#step-2-configure-route-rules)中选择**自动生成**，则不需要配置域名解析，域名会自动由 [nip.io](https://nip.io/) 解析为网关地址。
+
+若在[配置路由规则](#step-2-configure-route-rules)中选择**指定域名**，则需要在 DNS 服务器配置域名解析，或者在客户端机器上将`<路由网关地址> <路由域名>`添加到  `etc/hosts` 文件。
+
+## 访问应用路由
+
+### NodePort 访问方式
+
+1. 登录连接到应用路由网关地址的客户端机器。
+
+2. 使用`<路由域名>:<NodePort>/<服务路径>`地址访问应用路由的后端服务。
+
+   ![access-route-nodeport](/images/docs/zh-cn/project-user-guide/application-workloads/routes/access-route-nodeport.png)
+
+### LoadBalancer 访问方式
+
+1. 登录连接到应用路由网关地址的客户端机器。
+
+2. 使用`<路由域名>/<服务路径>`地址访问应用路由的后端服务。
+
+   ![access-route-loadbalancer](/images/docs/zh-cn/project-user-guide/application-workloads/routes/access-route-loadbalancer.png)
+
+{{< notice note >}}
+
+如果您需要使用 NodePort 或 LoadBalancer 从私有网络外部访问应用路由，具体取决于您的网络环境：
+
+* 您可能需要在基础设施环境中配置流量转发和防火墙规则，以便访问应用路由的网关地址和端口号。
+* 若在[配置路由规则](#step-2-configure-route-rules)中选择**自动生成**，则可能需要手动[编辑路由规则](#edit-the-route)将路由域名中的网关地址改为您私有网络的外部 IP 地址。
```

Comment:
Change the referred subtitle to Chinese. This is a Chinese document and all the subtitles are in Chinese.